app/controllers/Assessments.scala

```diff
@@ -95,12 +96,16 @@ class Assessments @Inject()(
         logger.warn("property link assessment call failed", e)
         val linkF = if (owner) propertyLinks.getMyOrganisationPropertyLink(submissionId) else propertyLinks.getMyClientsPropertyLink(submissionId)
         linkF.map {
-          case Some(link) => Redirect(routes.Assessments.viewSummary(link.uarn, true))
+          case Some(link) => Redirect(getViewSummaryCall(link.uarn, pending = true, owner))
```

Comment:
is there a reason we are not using reverse router?

Reply:
we are using the reverse router

Reply:
coolio.